app/assets/javascripts/models/user.js

```diff
@@ -1,3 +1,15 @@
 module.exports = Backbone.Model.extend({
-  name: 'user'
+  name: 'user',
+  initialize: function() {
+  },
+  getCurrent: function(callback) {
+    var options = {
+      type: 'GET',
+      dataType: 'json',
+      url: '/users/current',
+      success: callback,
+    };
+
+    return $.ajax(options);
```

Comment:
What do you think about using a promise here? Then, on the call site, you'll just do `user.getCurrent().then(getCurrentUserSuccess)`, I think it's much cleaner and avoid the creation of a parameter for the callback 😄